views/login.ejs

```diff
@@ -0,0 +1,43 @@
+<!-- Based on designs by Jason Zigrino (https://dribbble.com/JasonZigrino) and Himateja (https://codepen.io/Himateja) -->
+<head>
+	<link href="login.css" rel="stylesheet" type="text/css">
+</head>
+
+<!-- Based on designs by Jason Zigrino (https://dribbble.com/JasonZigrino) and Himateja (https://codepen.io/Himateja) -->
+<header>
+    <img src="https://cdn.rawgit.com/o2r-project/o2r-platform/master/client/img/o2r-logo-only-white.svg" />
+</header>
+<div class="login">
+    <div class="photo">
+    </div>
+    <span>Sign in to a test account using OAuth2</span>
+
+    <div id="login-form">
+        <form action="/login" method="post">
```

Comment:
Please create these three forms dynamically on the server-side using `config.testUsers`.